screenshot-iframe.js

```diff
@@ -0,0 +1,34 @@
+import {storageHost} from './constants';
+
+export const getScreenshotBlob = async (reqUrl, hash, ext, type, width, height) => {
```

Comment:
you are assuming here that hash is the second parameter this means if the user has to pass an argument for hash 
he will using like this?

getScreenshotBlob(undefined, 'some hash') ?
it looks not good to me 

Check for the first paramter whether it's a URL or hash and based on that construct the reqUrl

Reply:
Got it, fixed.